Packs/AzureSentinel/Scripts/MicrosoftSentinelSubmitNewComment/MicrosoftSentinelSubmitNewComment.py

```diff
@@ -0,0 +1,73 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+def add_new_comment(context_results: dict):
+    """
+    Adds a new comment to an Azure Sentinel incident and returns the result.
+
+    Args:
+        context_results (dict): Contains incident details, including "CustomFields" with "sourceid" for the incident ID
+                                and "sourceInstance" for the instance name.
+
+    Returns:
+        CommandResults: Includes a markdown-formatted string confirming the new comment and the comment details.
+    """
+    args = demisto.args()
+
+    incident_id = dict_safe_get(
+        context_results, ["CustomFields", "sourceid"], ""
+    ) or args.get("incident_id")
+    mirror_dir = next(
+        (
+            tag["value"]
+            for tag in context_results.get("labels", [])
+            if tag.get("type") == "mirror_direction"
+        ),
+        context_results.get("dbotMirrorDirection"),
+    )
+    instance_name = context_results.get("sourceInstance") or args.get("using")
+    new_comment = args.get("new_comment")
+    if mirror_dir in ["null", "Out"]:
+        return CommandResults(
+            readable_output="""The comment has not been added.
+            Comments can only be added when the Mirroring Direction is set to 'Incoming' or 'Incoming and Outgoing'."""
+        )
+    if not incident_id:
+        return_error(
+            "Incident ID not found. Please provide the remote 'incident_id' either as an argument when running the script from the War Room."  # noqa: E501
+        )
+
+    demisto.debug(f"update remote incident with new XSOAR comments: {new_comment}")
+    execute_command(
+        "azure-sentinel-incident-add-comment",
+        {"using": instance_name, "incident_id": incident_id, "message": new_comment},
+    )
+    readable_output = tableToMarkdown(
+        "The new comment has been recorded and will appear in your comments field in a minute \n(Only if you have A 'Mirror In')",  # noqa: E501
```

Comment:
```suggestion
        ""The new comment has been recorded and will appear in your comments field shortly. Note: This will only occur if you have the 'Mirror In' option enabled."",  # noqa: E501
```